libfwupdplugin/meson.build

```diff
@@ -203,6 +203,8 @@ if get_option('introspection') and get_option('gusb')
     includes : [
       'Gio-2.0',
       'GObject-2.0',
+      'GUsb-1.0',
+#      'Xmlb-2.0',
```

Comment:
drop it instead of comment out?

Reply:
Again, we need it, but we need to depend on a version of libxml that has this bug fixed: https://github.com/hughsie/libxmlb/pull/90

Reply:
I merged a better version of this in https://github.com/fwupd/fwupd/pull/3283